### PR TITLE
Fix to correctly select property

### PR DESCRIPTION
**Pull request by sunglim**
- Adding 'assert.js' that has asser functions.
- Fix wrongly selected value.

### CODE REVIEW

**Comment by sunglim** [comment]
@RandomlyKnighted Can you take a look?

On second thought. I decided not to use jquery. let's be simple.

File: app/js/index.js [hunk 1]
```diff
@@ -34,13 +36,14 @@ var str2ab = function(str) {
...
 
-var getIndexByValue = function( element, value ) {
+var getIndexByValue = function(element, value) {
   var list = element.options;
   for(var i = 0; i < list.length; i++) {
     if (list[i].value === value) {
       return i;
     }
   }
+  assertNotReached();
```
Comment (RandomlyKnighted) [inline app/js/index.js]: Why are you using assert.js in the production code? I've always seen asserts used as part of a unit test, which are usually in a test folder. This way you can test the code during development? It looks like the only thing you're using from that is `assertNotReached()`. Why not just throw an error or an exception?

File: app/js/index.js [hunk 2]
```diff
@@ -149,8 +153,9 @@ document.querySelector("#connectBtn").addEventListener("click", function(event)
     return;
 
     // Get the serial port (i.e. COM1, COM2, COM3, etc.)
-    var portElement = document.querySelector("#portDropdown");
-    var port = portElement.options[portElement.selectedIndex].value;
+    //var portElement = document.querySelector("#portDropdown");
+    //var port = portElement.options[portElement.selectedIndex].value;
+    var port = "COM1";
```
Comment (RandomlyKnighted) [inline app/js/index.js]: Why are we hard-coding the port?

File: app/js/index.js (hunk 1 above)
Comment (sunglim) [inline app/js/index.js]: Thank you for the comment.
When I play with Chromium I saw some `assert` in production code.[1]

But after second thought, You're right. We should not use assert in production code.
Can you please remove this line and assert.js?  When I'm ready to disable assert code when packing. I'll write it again.

[1] https://cs.chromium.org/search/?q=assert+file:%5Esrc/ui/webui/resources/js/&sq=package:chromium&type=cs

Comment (RandomlyKnighted) [inline app/js/index.js]: Sure I'll remove them. Other than that it seemed good. Thanks!

Comment (sunglim) [inline app/js/index.js]: Woot! Woot! Thank you for the review!

**Pull request merged**